modules/phase_field/src/kernels/ParamReaction.C

```diff
@@ -0,0 +1,34 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "ParamReaction.h"
+
+template <>
+InputParameters
+validParams<ParamReaction>()
+{
+  InputParameters params = validParams<Kernel>();
+  params.addRequiredParam<Real>("param", "Parameter multiplier");
```

Comment:
I suggest we rename this parameter `coefficient` (just like it is named in CoefReaction in moose test) if you end up not moving the test kernel to framework now. That way it will be less painful if we decide to  consolidate the two copies into one (no input file modifications).